### DISCUSSION TITLE
turbo gen can not specify other folder (not apps and packages)

### DISCUSSION THREAD

**Discussion by kin-for-test** [vercel/turborepo#5664] [Help] [answered]
my repo like this:

repo
|___apps
|___packages
|___util

when i use turbo gen workspaces  ,there only apps and packages but missing util ,how can i choose util folder ?

**Comment by anthonyshew** [comment, marked as answer]
I've created an issue from this. Please follow along there!

https://github.com/vercel/turbo/issues/6763
